File: 백준/Bronze/25881. Electric Bill/Electric Bill.py
import sys
input=lambda:sys.stdin.readline().rstrip()
gen,add=map(int,input().split())
for _ in range(int(input())):
    consu=int(input())
    print(consu,consu*gen if consu<=1000 else 1000*gen+(consu-1000)*add)

File: 백준/Bronze/25881. Electric Bill/Electric Bill.cc
#include <iostream>
using namespace std;
int main(void) {
    int gen, add, n, consu;
    cin >> gen >> add;
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> consu;
        cout << consu << ' ' << (consu <= 1000 ? gen * consu : 1000 * gen + (consu - 1000) * add) << "\n";
    }
    return 0;
}